### PR TITLE
Convert InstallLcmapsVoms

### PR DESCRIPTION
**Pull request by matyasselmeci**
https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/InstallLcmapsVoms
- [ ] Add date to spreadsheet
- [ ] Replace twiki doc with link

### CODE REVIEW

File: docs/common/install-lcmaps-voms.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+Configuring the LCMAPS VOMS Plugin
+----------------------------------
+
+The following section describes the steps required to configure the LCMAPS VOMS plugin for authentication. If you are using OSG 3.3 packages, there are software-specific instructions that must be followed for [HTCondor-CE](Documentation.Release3/InstallHTCondorCE), [GridFTP](Documentation.Release3/InstallOSGGridFTP), and [XRootD](Documentation.Release3/InstallXrootd). To check if you are running OSG 3.3, run the following command:
+
+``` console
+[root@server]# rpm -q --queryformat="%{VERSION}\n" osg-release
+```
+
+Additionally, there is [optional configuration](#OptionalConfig) if you need to make changes to the default mappings.
```
Comment (brianhlin) [inline docs/common/install-lcmaps-voms.md]: To link sections within a page, lowercase the entire section name and replace spaces with dashes. If there are multiple sections with the same name you can link the subsequent sections by appending `_N` where `N` is the section's ordinal number minus one, e.g. append `_1` for the second section. For example, if you have three sections named "Optional Configuration", link them like so:

```
[1st section](#optional-configuration)
[2nd section](#optional-configuration_1)
[3rd section](#optional-configuration_2)
```

File: docs/common/install-lcmaps-voms.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+``` ini
+glexec_location = UNAVAILABLE
+edit_lcmaps_db = True
+authorization_method = vomsmap
+```
+
+
+### Supporting mapped VOs and users
+
+Unix accounts must exist for each VO, VO role, VO group, or user you choose to support in the [mapfiles](#RefConfigFiles):
```
Comment (brianhlin) [inline docs/common/install-lcmaps-voms.md]: Fix broken link

File: docs/common/install-lcmaps-voms.md [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+
+1.  Consult the default VO mappings in `/usr/share/osg/voms-mapfile-default` to determine the mapped Unix account names. Each of the mapfiles has the following format:
+
+        "%RED%<VO, VO role, VO group or user>%ENDCOLOR%" %RED%<Unix account>%ENDCOLOR%
+
+
+2.  Create Unix accounts for each VO, VO role, VO group, and user that you wish to support
+3.  Edit `/etc/osg/config.d/30-gip.ini` and specify the supported VOs per [Subcluster or ResourceEntry section](Documentation.Release3/IniConfigurationOptions#Subcluster_Resource_Entry_Config):
```
Comment (brianhlin) [inline docs/common/install-lcmaps-voms.md]: Broken link

File: docs/common/install-lcmaps-voms.md [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+
+``` ini
+allowed_vos="VO1,VO2..."
+```
+
+### Applying configuration settings
+
+Making changes to the OSG configuration files in the `/etc/osg/config.d` directory does not apply those settings to software automatically. For the OSG settings, use the [osg-configure](IniConfigurationOptions) tool to validate (to a limited extent) and apply the settings to the relevant software components. If instead you wish to manage the LCMAPS VOMS plugin configuration yourself, skip to the [manual configuration section](#RefManConfig).
```
Comment (brianhlin) [inline docs/common/install-lcmaps-voms.md]: Pair of rboken links

File: docs/common/install-lcmaps-voms.md [hunk 5]
```diff
@@ -0,0 +1,322 @@
...
+
+The following subsections contain information on migration from `edg-mkgridmap`, mapping or banning users by their certificates' Distinguished Names (DNs) or by their proxies' VOMS attributes. Any optional configuration is to be performed after the installation and configuration sections above.
+
+For a table of the configuration files and their order of evaluation, consult the [reference section](#RefConfigFiles).
+
+-   [Migrating from edg-mkgridmap](#EdgmkgridmapMigration)
+-   [Mapping VOs](#OptionalMapVos)
+-   [Mapping users](#OptionalMapUsers)
+-   [Banning VOs](#OptionalBanVos)
+-   [Banning users](#OptionalBanUsers)
```
Comment (brianhlin) [inline docs/common/install-lcmaps-voms.md]: These links all need to be fixed

File: docs/common/install-lcmaps-voms.md [hunk 6]
```diff
@@ -0,0 +1,322 @@
...
+        :::console
+        [root@server]# osg-configure -c
+
+
+
+### Optional configuration
+
+The following subsections contain information on migration from `edg-mkgridmap`, mapping or banning users by their certificates' Distinguished Names (DNs) or by their proxies' VOMS attributes. Any optional configuration is to be performed after the installation and configuration sections above.
+
+For a table of the configuration files and their order of evaluation, consult the [reference section](#RefConfigFiles).
```
Comment (brianhlin) [inline docs/common/install-lcmaps-voms.md]: Another broken link

File: docs/common/install-lcmaps-voms.md [hunk 7]
```diff
@@ -0,0 +1,322 @@
...
+-   [Banning users](#OptionalBanUsers)
+
+
+#### Migrating from edg-mkgridmap
+
+The program edg-mkgridmap (found in the package `edg-mkgridmap`), used for authentication on HTCondor-CE, GridFTP, and XRootD hosts, is no longer available starting in OSG 3.4. The LCMAPS VOMS plugin (package `lcmaps-plugins-voms`) now provides the same functionality. To migrate from edg-mkgridmap to the LCMAPS VOMS plugin, perform the following procedure:
+
+1.  Configure user DN mappings:
+
+    * If you have a local grid mapfile (see [this section](Documentation.Release3/Edg-mkgridmap#3_0_Configuration)), replace the contents of `/etc/grid-security/grid-mapfile` with the contents of the local grid mapfile.
```
Comment (brianhlin) [inline docs/common/install-lcmaps-voms.md]: One more link

File: docs/common/install-lcmaps-voms.md [hunk 8]
```diff
@@ -0,0 +1,322 @@
...
+```
+
+and then restart `condor-ce`.
+
+Once you are satisfied that your mappings are working, you may remove this file and restart `condor-ce` in order to reduce the load on your CE caused by authentication.
+
+Getting Help
+------------
+
+To get assistance, please use the [this page](Documentation.HelpProcedure).
```
Comment (brianhlin) [inline docs/common/install-lcmaps-voms.md]: This page has been migrated and can be changed to `../common/help`

File: docs/common/install-lcmaps-voms.md [hunk 9]
```diff
@@ -0,0 +1,322 @@
...
+|:--------------------------------------|:---------|:------------------|
+| `/etc/grid-security/ban-mapfile`      | Admin    | Ban DNs           |
+| `/etc/grid-security/ban-voms-mapfile` | Admin    | Ban VOs           |
+| `/etc/grid-security/grid-mapfile`     | Admin    | Map DNs           |
+| `/etc/grid-security/voms-mapfile`     | Admin    | Map VOs           |
+| `/usr/share/osg/voms-mapfile-default` | OSG      | Map VOs (default) |
+
+### Manual Configuration
+
+This section is intended for use as reference if you choose to forego configuring the LCMAPS VOMS plugin via osg-configure (i.e., if you prefer a configuration management system like [Ansible](https://www.ansible.com/) or [Puppet](https://puppet.com/)). Therefore, the following instructions serve as a replacement for [this section](#ApplyConfig).
```
Comment (brianhlin) [inline docs/common/install-lcmaps-voms.md]: Broken link

**Comment by matyasselmeci** [comment]
Can you take another look?

**Comment by brianhlin** [comment]
LGTM. Go ahead and merge once the conflicts are sorted out

**Pull request merged**